test/lit.cfg

```diff
@@ -374,6 +381,10 @@ if run_os == 'openbsd' and run_cpu == 'amd64':
    target_arch = run_cpu
    run_cpu = 'x86_64'
 
+if run_os == 'freebsd' and run_cpu == 'amd64':
```

Comment:
I know you are just cargoculting what openbsd did here, but can you write a comment here explaining why we are doing this? It would just be helpful to document.